skorch/tests/callbacks/test_lr_scheduler.py

```diff
@@ -186,6 +186,47 @@ def test_lr_scheduler_set_params(self, classifier_module, classifier_data):
         net.fit(*classifier_data)  # we need to trigger on_train_begin
         assert net.callbacks[0][1].lr_scheduler_.base_lrs[0] == 456
 
+    @pytest.mark.parametrize('policy,kwargs', [
+        (StepLR, {'gamma': 0.9, 'step_size': 1})
+    ])
+    def test_lr_scheduler_record_epoch_step(self,
+                                            classifier_module,
+                                            classifier_data,
+                                            policy,
+                                            kwargs):
+        epochs = 3
+        scheduler = LRScheduler(policy, **kwargs)
+        lrs = scheduler.simulate(epochs, initial_lr=123.)
+        net = NeuralNetClassifier(
+            classifier_module,
+            max_epochs=epochs,
+            lr=123,
+            callbacks=[('scheduler', scheduler)]
+        )
+        net.fit(*classifier_data)
+        if hasattr(scheduler.lr_scheduler_, "get_last_lr"):
```

Comment:
As stated by @BenjaminBossan , I would prefer a `@pytest.mark.skip` here based on `torch.__version__` then checking for the method here.

Reply:
ping @guybuk

Reply:
Never actually used `pytest` before. Is this what you mean?
```python
@pytest.mark.skipif(version.parse(torch.__version__)<version.parse("1.4.0")
def test...
```

Reply:
```py
from distutils.version import LooseVersion

@pytest.mark.skipif(LooseVersion(torch.__version__) < '1.4')
def test...
```